apps/src/aiTutor/constants.ts

```diff
@@ -3,11 +3,7 @@ import {
   AITutorInteractionStatus as Status,
 } from '@cdo/apps/aiTutor/types';
 
-export const compilationSystemPrompt =
-  'You are a tutor in a high school computer science class. Students in the class are studying Java and they would like to know in age-appropriate, clear language why their code does not compile. Do not write any code.';
-export const validationSystemPrompt =
-  'You are a tutor in a high school computer science class. Students in the class are studying Java and they would like to know in age-appropriate, clear language why their tests are not passing. Do not write any code.';
-export const generalChatSystemPrompt =
+export const systemPrompt =
```

Comment:
I don't think we need three different system prompts here. Instead of having three different "tutors," we can just use the same tutor and ask it, "Why doesn't my code compile?" or, "Why aren't my tests passing?"